lib/fastlane/actions/taxi.rb

```diff
@@ -0,0 +1,79 @@
+
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class TaxiAction < Action
```

Comment:
I think we should name it more of what it's actually doing. How about `install_on_device`?